UI/src/app/components/comfy/comfy.component.html

```diff
@@ -30,7 +30,7 @@ <h3 class="filter-label">Link tables</h3>
         (reset)="filterByNameReset('source', $event)"
       ></app-comfy-search-by-name>
       <h4 class="page-column-label">Source tables</h4>
-      <div class="scrolled-items">
+      <div class="scrolled-items" style="direction:rtl;">
```

Comment:
add this style into scrolled-items

Reply:
Or create and use new class:
`.scrolled-items-rtl {
   @extend .scrolled-items;
   direction: rtl;
}`

Reply:
Fixed in commit #6afb96d